### DISCUSSION TITLE
Check RedHat 8 Enterprise server compatibility.

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#9783] [Archive]
I tried to install on RH8 Server. 
I’ve a lot’s of errors to install Directus because some RH packages are not compatible with latest Node and Directus packs.
Could you check?
Thx.

**Comment by azrikahar** [comment]
Perhaps you can provide more details, like an error log? "Some RH packages" can be pretty vague here. As for Node version, to make sure it's at least v12.20 as well 👍 Not sure what's the provided version in RH8 so at least that's another possible thing we can attempt to rule out here once you confirmed it.

**Comment by biapar** [comment]
I installed Node v12 and v14. I've to rebuild the server and send the logs.

**Reply by azrikahar** [reply]
Thanks for the update on the Node version, but I'd also recommend to verify the minor version for v12, since if it's anything lower than `12.20`, such as `12.3`, `12.4` etc, it is also not supported.
